pkgs/tools/security/bitwarden_rs/vault.nix

```diff
@@ -23,6 +23,6 @@ stdenv.mkDerivation rec {
     homepage = "https://github.com/dani-garcia/bw_web_builds";
     platforms = platforms.all;
     license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
@Mic92 Where did you get that information from?